x-pack/legacy/plugins/siem/public/components/timeline/body/column_headers/index.tsx

```diff
@@ -8,12 +8,13 @@ import { noop } from 'lodash/fp';
 import * as React from 'react';
 import { Draggable, Droppable } from 'react-beautiful-dnd';
 
+import { EuiCheckbox } from '@elastic/eui';
```

Comment:
nit - let's put it on the top of the imports

Reply:
Now that https://github.com/benmosher/eslint-plugin-import/pull/1105 is merged I think we're good to turn on the grouping/alphabetize linter and make this a thing of the past 🙂